linux/xdg-desktop-portal/build.gradle.kts

```diff
@@ -0,0 +1,88 @@
+plugins {
+    java
+    id("dev.nokee.jni-library")
+    id("dev.nokee.cpp-language")
+    `uber-jni-jar`
+    `use-prebuilt-binaries`
+    kotlin("jvm")
+    kotlin("kapt")
+}
+
+dependencies {
+    implementation(projects.autoDarkModeBase)
+    implementation(libs.darklaf.nativeUtils)
+
+    // TODO add xdg-desktop-portal dependency
+    implementation("com.github.hypfvieh:dbus-java:3.3.1")
+    implementation("com.github.hypfview:dbus-java-transport-jnr-unixsocket:4.0.0")
```

Comment:
Those dependencies don't seem to exists according to CI

Reply:
My IDE can find dbus-java, but not the second one. It's in the [dbus-java repository](https://github.com/hypfvieh/dbus-java), so I just commented it out for now. However, I don't know if that works since I can't build it yet...